bukkit/src/main/resources/plugin.yml

```diff
@@ -62,3 +67,7 @@ permissions:
         description: 'Label others as cracked'
         children:
             ${project.artifactId}.command.cracked: true
+
+    ${project.artifactId}.command.delete:
+        description: 'Delete other players profile data'
+        default: true
```

Comment:
I guess this shouldn't be true right ;)

A normal player shouldn't be able to delete data.